docs/retrieverjl.rst

```diff
@@ -0,0 +1,131 @@
+==========================
+Data Retriever using Julia
+==========================
+
+The wrapper module for `Data Retriever <http://data-retriever.org>`_ has been implemented as `Retriever.jl <https://github.com/weecology/retriever.jl.git>`_. All the functions work and feel the same as the python Retriever module. The module has been created using ``PyCall`` hence all the functions are analogous to the functions of retriever python module.
+
+
+Installation
+============
+
+To use Julia package ``retriever.jl``, you first need to `install Retriever <introduction.html#installing-binaries>`_.
+
+After installing Data Retriever, ``PyCall`` package needs to be installed.
+
+.. code-block:: julia
+
+    """ Julia Command """
+
+    Pkg.install("PyCall")
+
+The package hasn't been registered in official Julia packages. So each time you want to use Retriever.jl you will have to follow this step.
```

Comment:
`The package hasn't been registered in official Julia packages. So each time you want to use Retriever.jl you will have to follow this step.` change this to 
To install from Source, download or checkout the source from the `github page <https://github.com/weecology/retriever.jl.git>`_.